src/main/resources/liquibase/update/20190220_create_classifier_table_and_content.sql

```diff
@@ -0,0 +1,106 @@
+
+INSERT INTO riha.classifier (id, type, code, value, discriminator) VALUES (nextval('riha.classifier_seq'), 'system_check_status', 'CANCELLED', 'CANCELLED', 'TEXT');
+INSERT INTO riha.classifier (id, type, code, value, discriminator) VALUES (nextval('riha.classifier_seq'), 'system_check_status', 'PENDING', 'PENDING', 'TEXT');
+INSERT INTO riha.classifier (id, type, code, value, discriminator) VALUES (nextval('riha.classifier_seq'), 'system_check_status', 'IN_PROGRESS', 'IN_PROGRESS', 'TEXT');
+INSERT INTO riha.classifier (id, type, code, value, discriminator) VALUES (nextval('riha.classifier_seq'), 'system_check_status', 'FAILED', 'FAILED', 'TEXT');
+INSERT INTO riha.classifier (id, type, code, value, discriminator) VALUES (nextval('riha.classifier_seq'), 'system_check_status', 'PASSED', 'PASSED', 'TEXT');
+
+INSERT INTO riha.classifier (id, type, code, json_value, discriminator) VALUES (nextval('riha.classifier_seq'), 'access_restriction_reasons', 'CODE_18', '{"code": 18, "legislation": "AvTS § 35 lg 1 p 9", "description": "Teave turvasüsteemide, turvaorganisatsiooni või turvameetmete kirjelduse kohta"}', 'JSON');
+INSERT INTO riha.classifier (id, type, code, json_value, discriminator) VALUES (nextval('riha.classifier_seq'), 'access_restriction_reasons', 'CODE_19', '{"code": 19, "legislation": "AvTS § 35 lg 1 p 10", "description": "Tehnoloogilisi lahendusi sisaldav teave"}', 'JSON');
+INSERT INTO riha.classifier (id, type, code, json_value, discriminator) VALUES (nextval('riha.classifier_seq'), 'access_restriction_reasons', 'CODE_36', '{"code": 36, "legislation": "AvTS § 35 lg 1 p 17", "description": "Ärisaladus"}', 'JSON');
+INSERT INTO riha.classifier (id, type, code, json_value, discriminator) VALUES (nextval('riha.classifier_seq'), 'access_restriction_reasons', 'CODE_38', '{"code": 38, "legislation": "AvTS § 35 lg 1 p 18 (1)", "description": "Elutähtsa teenuse riskianalüüsi ja toimepidevuse plaani puudutav teave"}', 'JSON');
+INSERT INTO riha.classifier (id, type, code, json_value, discriminator) VALUES (nextval('riha.classifier_seq'), 'access_restriction_reasons', 'CODE_39', '{"code": 39, "legislation": "AvTS § 35 lg 2 p", "description": "Õigusaktide eelnõud enne nende kooskõlastamiseks saatmist või vastuvõtmiseks esitamist"}', 'JSON');
+INSERT INTO riha.classifier (id, type, code, json_value, discriminator) VALUES (nextval('riha.classifier_seq'), 'access_restriction_reasons', 'CODE_40', '{"code": 40, "legislation": "AvTS § 35 lg 2 p 2", "description": "Dokumendi kavand ja selle juurde kuuluvad dokumendid"}', 'JSON');
+
+INSERT INTO riha.classifier (id, type, code, value, discriminator) VALUES (nextval('riha.classifier_seq'), 'topics_that_do_not_require_feedback_on_creation', 'X_ROAD_SUBSYSTEM', 'x-tee alamsüsteem', 'TEXT');
+INSERT INTO riha.classifier (id, type, code, value, discriminator) VALUES (nextval('riha.classifier_seq'), 'topics_that_do_not_require_feedback_on_creation', 'STANDARD_SOLUTION', 'standardlahendus', 'TEXT');
```

Comment:
please remove 'standardlahendus' classifier, as it is deleted as part of other task